vector/apps/web/lib/testing/TESTING_ENVIRONMENT.md

```
@@ -0,0 +1,1027 @@
+    "test:agent:scenarios": "ts-node scripts/test-agent.ts --filter=Create"
+  }
+}
+```
+
+---
+
+## Output Examples
+
+### Terminal Output (Normal)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)